SpatialGDK/Source/SpatialGDK/Public/Utils/EntityPool.h

```diff
@@ -45,18 +42,13 @@ class SPATIALGDK_API UEntityPool : public UObject
 	void Advance();
 
 private:
-	void OnEntityRangeExpired(uint32 ExpiringEntityRangeId);
-
 	UPROPERTY()
 	USpatialNetDriver* NetDriver;
 
-	FTimerManager* TimerManager;
 	TArray<EntityRange> ReservedEntityIDRanges;
 
-	bool bIsReady;
-	bool bIsAwaitingResponse;
-
-	uint32 NextEntityRangeId;
+	bool bIsReady = false;
```

Comment:
I wasn't sure whether this would be 0-initialised by default, so I asked in the cpp channel. It's possible (even likely!) I've misunderstood the response, so I'm not 100% on these default initialisers.

Reply:
@samiwh for more informed opinion, but I guess since we have not seen this break, I think, they probably are always zero-initialized.

Reply:
https://en.cppreference.com/w/cpp/language/default_initialization
condition 3) and the third bullet point down

tl;dr this change is correct; we should explicitly initialising this. I'm wondering if this might have caused some bugs that have appeared recently.

Reply:
Ah just read the slack thread which says basically that.

> It's possible (even likely!) I've misunderstood

No you got it right :)

Reply:
do we have a bias for doing initialisation in the member initializer list in the contructor, or do we mix and match?